scripts/travis/install_elasticsearch5.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sudo apt-get autoremove --purge elasticsearch
+wget -qO - https://packages.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
+echo "deb http://packages.elastic.co/elasticsearch/5.x/debian stable main" | sudo tee -a /etc/apt/sources.list.d/elk.list
```

Comment:
These look like the wrong URLs. From https://www.elastic.co/guide/en/elasticsearch/reference/current/deb.html#deb-repo:

```
wget -qO - https://artifacts.elastic.co/GPG-KEY-elasticsearch | sudo apt-key add -
echo "deb https://artifacts.elastic.co/packages/5.x/apt stable main" | sudo tee -a /etc/apt/sources.list.d/elastic-5.x.list
```